build fails

The latest version in the master branch cannot compile. Could anyone fix this problem?

```
 In file included from ../../src/backend/networking/peloton_endpoint.h:16:0,
                 from ../../src/backend/networking/rpc_client.h:17,
                 from ../../src/backend/networking/rpc_channel.cpp:14:
../../src/backend/networking/peloton_service.h:97:19: error: 'QueryPlanExecRequest' does not name a
 type
             const QueryPlanExecRequest* request,
                   ^
../../src/backend/networking/peloton_service.h:97:41: error: ISO C++ forbids declaration of 'reques
t' with no type [-Werror=permissive]
             const QueryPlanExecRequest* request,
                                         ^
../../src/backend/networking/peloton_service.h:98:13: error: 'QueryPlanExecResponse' has not been d
eclared
             QueryPlanExecResponse* response,
             ^
In file included from ../../src/backend/networking/rpc_channel.cpp:14:0:
../../src/backend/networking/rpc_client.h:84:24: error: 'QueryPlanExecRequest' does not name a type
   void QueryPlan(const QueryPlanExecRequest* request,
                        ^
../../src/backend/networking/rpc_client.h:84:46: error: ISO C++ forbids declaration of 'request' wi
th no type [-Werror=permissive]
   void QueryPlan(const QueryPlanExecRequest* request,
                                              ^
../../src/backend/networking/rpc_client.h:85:18: error: 'QueryPlanExecResponse' has not been declar
ed
                  QueryPlanExecResponse* response);
                  ^
cc1plus: all warnings being treated as errors
```

--- a/src/backend/networking/Makefile.am
+++ b/src/backend/networking/Makefile.am
@@ -15,7 +15,7 @@ MOSTLYCLEANFILES = \
 				    backend/networking/abstract_service.pb.h \
 				    backend/networking/queryplan.pb.h 
 
-backend/networking/abstract_service.pb.cc:
+backend/networking/abstract_service.pb.cc: $(srcdir)/backend/networking/abstract_service.proto
 	$(PROTOC) --proto_path=$(srcdir)/backend/networking \
 	$(srcdir)/backend/networking/abstract_service.proto \
 	--cpp_out=$(srcdir)/backend/networking

